2019-03-01 20:58:06,377 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 20:58:06,378 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 20:58:06,378 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 20:58:15,448 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 20:58:15,450 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 20:58:15,451 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 21:12:33,149 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 21:12:33,150 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 21:12:33,150 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 21:15:30,162 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 21:15:30,162 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 21:15:30,163 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 21:17:55,122 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 21:17:55,122 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 21:17:55,122 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 21:20:42,971 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 21:20:42,972 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 21:20:42,972 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 21:28:02,456 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 21:28:02,456 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 21:28:02,456 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 21:31:31,205 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 21:31:31,205 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 21:31:31,205 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 21:42:43,455 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 21:42:43,455 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 21:42:43,455 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 21:49:23,147 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 21:49:23,147 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 21:49:23,148 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 21:50:57,993 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 21:50:57,994 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 21:50:57,994 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 21:53:05,508 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 21:53:05,508 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 21:53:05,509 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 21:54:45,688 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 21:54:45,689 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 21:54:45,689 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 22:12:09,766 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 22:12:09,766 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 22:12:09,766 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 22:20:34,659 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 22:20:34,660 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 22:20:34,660 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 22:22:34,225 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 22:22:34,225 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 22:22:34,225 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 22:24:29,389 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 22:24:29,390 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 22:24:29,390 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 22:35:21,948 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 22:35:21,949 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 22:35:21,949 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 22:36:53,408 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 22:36:53,409 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 22:36:53,409 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 22:55:08,824 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 22:55:08,825 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 22:55:08,825 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 22:57:29,709 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 22:57:29,710 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 22:57:29,710 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 23:00:07,203 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 23:00:07,204 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 23:00:07,205 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 23:00:25,405 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 23:00:25,406 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 23:00:25,406 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 23:02:33,923 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 23:02:33,924 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 23:02:33,924 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 23:05:56,563 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 23:05:56,563 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 23:05:56,563 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 23:30:14,683 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 23:30:14,684 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 23:30:14,684 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 23:30:16,396 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:107]
2019-03-01 23:30:17,476 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:107]
2019-03-01 23:30:18,499 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:107]
2019-03-01 23:30:19,634 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:107]
2019-03-01 23:30:20,691 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:107]
2019-03-01 23:30:21,809 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:107]
2019-03-01 23:30:22,815 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:107]
2019-03-01 23:30:24,001 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:107]
2019-03-01 23:32:32,104 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 23:32:32,105 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 23:32:32,105 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 23:32:33,768 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:108]
2019-03-01 23:32:35,050 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:32:36,192 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:32:37,421 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:32:38,583 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:32:39,756 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:32:40,966 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:108]
2019-03-01 23:32:42,154 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:32:43,325 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:32:44,546 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:32:45,702 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:32:46,842 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:108]
2019-03-01 23:34:37,615 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 23:34:37,615 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 23:34:37,616 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 23:34:39,565 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:108]
2019-03-01 23:34:40,994 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:34:42,314 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:34:43,471 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:34:44,732 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:34:45,942 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:34:47,203 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:108]
2019-03-01 23:34:48,555 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:34:49,735 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:34:50,876 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:34:52,114 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:103]
2019-03-01 23:34:53,324 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:108]
2019-03-01 23:35:32,336 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 23:35:32,337 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 23:35:32,337 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 23:35:34,050 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:110]
2019-03-01 23:35:35,272 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:35:36,422 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:35:37,629 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:35:38,819 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:35:40,008 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:35:41,287 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:110]
2019-03-01 23:35:42,427 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:35:43,589 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:35:44,871 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:35:46,038 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:35:47,230 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:110]
2019-03-01 23:37:55,547 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 23:37:55,548 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 23:37:55,548 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 23:37:57,262 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:110]
2019-03-01 23:37:58,446 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:37:59,608 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:110]
2019-03-01 23:38:00,886 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:110]
2019-03-01 23:38:02,057 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:38:03,175 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:38:04,394 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:38:05,516 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:38:06,722 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:38:07,936 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:38:09,076 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:38:10,245 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:110]
2019-03-01 23:39:10,230 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 23:39:10,231 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 23:39:10,231 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 23:39:11,813 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:110]
2019-03-01 23:39:13,002 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:39:14,185 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:110]
2019-03-01 23:39:15,451 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:110]
2019-03-01 23:39:16,633 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:39:17,811 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:39:49,985 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 23:39:49,985 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 23:39:49,985 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 23:39:51,621 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:110]
2019-03-01 23:39:52,860 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:39:54,049 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:39:55,262 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:39:56,393 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:39:57,566 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:39:58,845 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:40:00,015 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:40:01,293 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:40:02,548 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:40:03,739 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:105]
2019-03-01 23:40:04,921 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:110]
2019-03-01 23:48:05,759 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的bindBankCard数据 - [do_testcase_excel.py:41]
2019-03-01 23:48:05,759 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 23:48:05,759 - test_bindBankCard - INFO - 读取bindBankCard所有测试用例成功 - [test_bindBankCard.py:27]
2019-03-01 23:48:05,840 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的sendMCode数据 - [do_testcase_excel.py:41]
2019-03-01 23:48:05,840 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 23:48:05,840 - test_sendMCode - INFO - 读取sendMCode所有测试用例成功 - [test_sendMCode.py:25]
2019-03-01 23:48:05,904 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的userRegister数据 - [do_testcase_excel.py:41]
2019-03-01 23:48:05,904 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 23:48:05,904 - test_userRegister - INFO - 读取userRegister所有测试用例成功 - [test_userRegister.py:29]
2019-03-01 23:48:05,949 - get test data - INFO - 开始读取D:\PYworkspace\webservice-api-test\data\webservice-testdata.xlsx中的verifyUserAuth数据 - [do_testcase_excel.py:41]
2019-03-01 23:48:05,949 - get test data - INFO - 读取完毕 - [do_testcase_excel.py:57]
2019-03-01 23:48:05,949 - test_verifyUserAuth - INFO - 读取verifyUserAuth所有测试用例成功 - [test_verifiedUserAuth.py:28]
2019-03-01 23:48:07,669 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:114]
2019-03-01 23:48:08,896 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:110]
2019-03-01 23:48:10,108 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:110]
2019-03-01 23:48:11,334 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:110]
2019-03-01 23:48:12,470 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:110]
2019-03-01 23:48:13,613 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:110]
2019-03-01 23:48:14,822 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:110]
2019-03-01 23:48:16,004 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:110]
2019-03-01 23:48:17,318 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:110]
2019-03-01 23:48:18,682 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:110]
2019-03-01 23:48:19,969 - test_bindBankCard - INFO - 状态码校验成功 - [test_bindBankCard.py:110]
2019-03-01 23:48:21,285 - test_bindBankCard - ERROR - 数据校验失败 - [test_bindBankCard.py:114]
2019-03-01 23:48:21,629 - test_sendMCode - INFO - 请求发送成功 - [test_sendMCode.py:54]
2019-03-01 23:48:21,889 - test_sendMCode - ERROR - 系统错误 - [test_sendMCode.py:57]
2019-03-01 23:48:22,226 - test_sendMCode - ERROR - 系统错误 - [test_sendMCode.py:57]
2019-03-01 23:48:22,515 - test_sendMCode - ERROR - 系统错误 - [test_sendMCode.py:57]
2019-03-01 23:48:22,890 - test_sendMCode - INFO - 请求发送成功 - [test_sendMCode.py:54]
2019-03-01 23:48:23,203 - test_sendMCode - ERROR - 系统错误 - [test_sendMCode.py:57]
2019-03-01 23:48:23,494 - test_sendMCode - ERROR - 系统错误 - [test_sendMCode.py:57]
2019-03-01 23:48:24,424 - test_userRegister - INFO - (commResult){
   retCode = "0"
   retInfo = "ok"
 } - [test_userRegister.py:68]
2019-03-01 23:48:24,424 - test_userRegister - INFO - select * FROM user_db.t_user_info WHERE Fuser_id = '余燕8 By夜雨声烦' - [test_userRegister.py:72]
2019-03-01 23:48:24,462 - test_userRegister - INFO - 状态码校验成功 - [test_userRegister.py:79]
2019-03-01 23:48:24,462 - test_userRegister - INFO - 数据校验成功 - [test_userRegister.py:83]
2019-03-01 23:48:24,771 - test_userRegister - INFO - (commResult){
   retCode = "19001"
   retInfo = "手机号错误"
 } - [test_userRegister.py:68]
2019-03-01 23:48:24,772 - test_userRegister - INFO - select * FROM user_db.t_user_info WHERE Fuser_id = '李旭4 By夜雨声烦' - [test_userRegister.py:72]
2019-03-01 23:48:24,805 - test_userRegister - INFO - 状态码校验成功 - [test_userRegister.py:79]
2019-03-01 23:48:24,805 - test_userRegister - INFO - 数据校验成功 - [test_userRegister.py:94]
2019-03-01 23:48:25,045 - test_userRegister - INFO - (commResult){
   retCode = "19001"
   retInfo = "手机号错误"
 } - [test_userRegister.py:68]
2019-03-01 23:48:25,045 - test_userRegister - INFO - select * FROM user_db.t_user_info WHERE Fuser_id = '刘斌5 By夜雨声烦' - [test_userRegister.py:72]
2019-03-01 23:48:25,080 - test_userRegister - INFO - 状态码校验成功 - [test_userRegister.py:79]
2019-03-01 23:48:25,080 - test_userRegister - INFO - 数据校验成功 - [test_userRegister.py:94]
2019-03-01 23:48:25,864 - test_userRegister - INFO - (commResult){
   retCode = "0"
   retInfo = "ok"
 } - [test_userRegister.py:68]
2019-03-01 23:48:25,865 - test_userRegister - INFO - select * FROM user_db.t_user_info WHERE Fuser_id = '崔萍10 By夜雨声烦' - [test_userRegister.py:72]
2019-03-01 23:48:25,908 - test_userRegister - INFO - 状态码校验失败 - [test_userRegister.py:104]
2019-03-01 23:48:26,489 - test_userRegister - INFO - (commResult){
   retCode = "30010008"
   retInfo = "验证码错误"
 } - [test_userRegister.py:68]
2019-03-01 23:48:26,489 - test_userRegister - INFO - select * FROM user_db.t_user_info WHERE Fuser_id = '张海燕8 By夜雨声烦' - [test_userRegister.py:72]
2019-03-01 23:48:26,523 - test_userRegister - INFO - 状态码校验成功 - [test_userRegister.py:79]
2019-03-01 23:48:26,523 - test_userRegister - INFO - 数据校验成功 - [test_userRegister.py:94]
2019-03-01 23:48:26,770 - test_userRegister - INFO - (commResult){
   retCode = "19001"
   retInfo = "验证码错误"
 } - [test_userRegister.py:68]
2019-03-01 23:48:26,771 - test_userRegister - INFO - select * FROM user_db.t_user_info WHERE Fuser_id = '徐建军10 By夜雨声烦' - [test_userRegister.py:72]
2019-03-01 23:48:26,816 - test_userRegister - INFO - 状态码校验成功 - [test_userRegister.py:79]
2019-03-01 23:48:26,816 - test_userRegister - INFO - 数据校验成功 - [test_userRegister.py:94]
2019-03-01 23:49:28,545 - test_userRegister - INFO - (commResult){
   retCode = "0"
   retInfo = "ok"
 } - [test_userRegister.py:68]
2019-03-01 23:49:28,545 - test_userRegister - INFO - select * FROM user_db.t_user_info WHERE Fuser_id = '邹志强7 By夜雨声烦' - [test_userRegister.py:72]
2019-03-01 23:49:28,586 - test_userRegister - INFO - 状态码校验失败 - [test_userRegister.py:104]
2019-03-01 23:49:29,143 - test_userRegister - INFO - (commResult){
   retCode = "19001"
   retInfo = "用户UID错误"
 } - [test_userRegister.py:68]
2019-03-01 23:49:29,143 - test_userRegister - INFO - select * FROM user_db.t_user_info WHERE Fuser_id = '陈浩10 By夜雨声烦' - [test_userRegister.py:72]
2019-03-01 23:49:29,181 - test_userRegister - INFO - 状态码校验成功 - [test_userRegister.py:79]
2019-03-01 23:49:29,182 - test_userRegister - INFO - 数据校验成功 - [test_userRegister.py:94]
2019-03-01 23:49:30,454 - test_verifyUserAuth - INFO - {'uid': '100005898', 'true_name': '张玉梅', 'cre_id': '130729199505280054'} - [test_verifiedUserAuth.py:83]
2019-03-01 23:49:30,583 - test_verifyUserAuth - INFO - (commResult){
   retCode = "0"
   retInfo = "ok"
 } - [test_verifiedUserAuth.py:86]
2019-03-01 23:49:30,790 - test_verifyUserAuth - INFO - 判断校验码成功 - [test_verifiedUserAuth.py:93]
2019-03-01 23:49:30,790 - test_verifyUserAuth - INFO - 验库成功 - [test_verifiedUserAuth.py:97]
2019-03-01 23:49:31,995 - test_verifyUserAuth - INFO - {'uid': '100005899', 'true_name': '', 'cre_id': '540224196206292235'} - [test_verifiedUserAuth.py:83]
2019-03-01 23:49:32,076 - test_verifyUserAuth - INFO - (commResult){
   retCode = "19001"
   retInfo = "真实姓名错误"
 } - [test_verifiedUserAuth.py:86]
2019-03-01 23:49:32,268 - test_verifyUserAuth - INFO - 判断校验码成功 - [test_verifiedUserAuth.py:93]
2019-03-01 23:49:32,269 - test_verifyUserAuth - INFO - 验库成功 - [test_verifiedUserAuth.py:106]
2019-03-01 23:49:33,342 - test_verifyUserAuth - INFO - {'uid': '100005900', 'true_name': '黄雪', 'cre_id': ''} - [test_verifiedUserAuth.py:83]
2019-03-01 23:49:33,427 - test_verifyUserAuth - INFO - (commResult){
   retCode = "19001"
   retInfo = "身份证号错误"
 } - [test_verifiedUserAuth.py:86]
2019-03-01 23:49:33,622 - test_verifyUserAuth - INFO - 判断校验码成功 - [test_verifiedUserAuth.py:93]
2019-03-01 23:49:33,622 - test_verifyUserAuth - INFO - 验库成功 - [test_verifiedUserAuth.py:106]
2019-03-01 23:49:34,830 - test_verifyUserAuth - INFO - {'uid': '100005901', 'true_name': '刘凤兰', 'cre_id': '3207231996052937540'} - [test_verifiedUserAuth.py:83]
2019-03-01 23:49:34,919 - test_verifyUserAuth - INFO - (commResult){
   retCode = "19001"
   retInfo = "身份证号错误"
 } - [test_verifiedUserAuth.py:86]
2019-03-01 23:49:35,117 - test_verifyUserAuth - INFO - 判断校验码成功 - [test_verifiedUserAuth.py:93]
2019-03-01 23:49:35,118 - test_verifyUserAuth - INFO - 验库成功 - [test_verifiedUserAuth.py:106]
2019-03-01 23:49:36,070 - test_verifyUserAuth - INFO - {'uid': '111111111', 'true_name': '吴婷婷', 'cre_id': '130723193506054440'} - [test_verifiedUserAuth.py:83]
2019-03-01 23:49:36,156 - test_verifyUserAuth - INFO - (commResult){
   retCode = "20042002"
   retInfo = "用户不存在"
 } - [test_verifiedUserAuth.py:86]
2019-03-01 23:49:36,347 - test_verifyUserAuth - INFO - 判断校验码成功 - [test_verifiedUserAuth.py:93]
2019-03-01 23:49:36,347 - test_verifyUserAuth - INFO - 验库成功 - [test_verifiedUserAuth.py:106]
2019-03-01 23:49:37,304 - test_verifyUserAuth - INFO - {'uid': None, 'true_name': '李玉兰', 'cre_id': '410301197008102230'} - [test_verifiedUserAuth.py:83]
2019-03-01 23:49:37,394 - test_verifyUserAuth - INFO - (commResult){
   retCode = "19001"
   retInfo = "用户ID错误"
 } - [test_verifiedUserAuth.py:86]
2019-03-01 23:49:37,598 - test_verifyUserAuth - INFO - 判断校验码成功 - [test_verifiedUserAuth.py:93]
2019-03-01 23:49:37,598 - test_verifyUserAuth - INFO - 验库成功 - [test_verifiedUserAuth.py:106]
